calico_upgrade/pkg/calicoupgrade.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2016 Tigera, Inc. All rights reserved.
```

Comment:
nit: need to update all the license to 2017 later

Reply:
goddammit - i thought i had done that

Reply:
oh - clearly had not.  darn